# 这种错误是怎么回事？参加比赛的策略

2015-09-17 07:58 [SYS ERROR] handle_data函数运行失败，今天无法生成信号: handle_data() internal error line 42, in handle_data IndexError: index 300 is out of bounds for axis 1 with size 300